src/wwwroot/controllers/MainCtrl.js

```diff
@@ -24,7 +24,7 @@
     $rootScope.getFullName = auth.getFullName;
     $rootScope.getAccountId = auth.getAccountId;
 
-    $rootScope.addError = function (error, actionDescription, rejectionTitle, statusCode, errorCode, callback) {
+    $rootScope.addError = function (error, actionDescription, rejectionTitle, statusCode, errorCode, functionToCall, buttonText, callback) {
```

Comment:
Also, did you consider to use `callback` in place of adding a new function `functionToCall`?